packages/kit/src/components/ListView/TableList.tsx

```
@@ -0,0 +1,755 @@
+
+interface ISortButtonProps {
+  label: string;
+  iconName?: IKeyOfIcons;
+  onPress?: IXStackProps['onPress'];
+  align?: 'flex-start' | 'center' | 'flex-end';
+}
+
+const SortButton = memo(
+  ({ label, iconName, onPress, align = 'flex-start' }: ISortButtonProps) => (
+    <XStack
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)